jmix-dynattr/dynattr/dynattr.gradle

```diff
@@ -28,9 +28,12 @@ dependencies {
     testImplementation 'org.springframework:spring-test'
     testImplementation 'org.spockframework:spock-core'
     testImplementation 'org.spockframework:spock-spring'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api'
```

Comment:
We don't need this dependency if you'll use Spock

Reply:
Is there any example of implementing unit test for entity with bean validation engine?